mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule OpentelemetryAbsinthe.MixProject do
       {:absinthe, ">= 1.5.0", optional: true},
       {:jason, "~> 1.2"},
       {:opentelemetry_api, "~> 1.1"},
-      {:telemetry, "~> 0.4 or ~> 1.0.0"}
+      {:telemetry, "~> 0.4 or ~> 1.1.0"}
```

Comment:
Can we do
```suggestion
      {:telemetry, "~> 0.4 or ~> 1.0"}
```
instead?
`~> 1.1.0` means `< 1.2.0 and >= 1.1.0`
while
`~>1.0` would mean `< 2.0.0 and >= 1.0.0`

Reply:
Good point.  

Thanks.